Fixed #25586 -- Fixed possible table cell misalignment in admin's tabular inline.

blueyed opened the pull request:
This fixes the table body (tbody) to be shifted by one column to the
left.  While the width of td.original and th.original is set to 0 in the
CSS, th.original was not defined.

I cannot really believe that this was broken for so long, and therefore some non-default settings / behaviour might trigger it?!  And/or browser behaviour changed in that regard?  Or some JS got missing?

Anyway, the problem can be seen in both Firefox and Chromium for me.

blueyed commented:
A screenshot to illustrate it: the first column is empty, while the option list should be there, and is shifted to the right instead.

![2015-10-19-200754_425x196_scrot](https://cloud.githubusercontent.com/assets/9766/10586654/6d0044ee-769d-11e5-896e-c19b5d2419d2.png)

timgraham commented:
It looks to me like colspan="2" in that template is meant to account for that. Possibly the first field on the form is hidden in your case so `forloop.first` is `False` and we don't get colspan=2.

blueyed commented:
Thanks for your feedback, @timgraham.

I've updated my patch to also remove the colspan=2, and updated the commit message with more information how to trigger it.

timgraham commented:
merged in 1745aa000a4e859c21270371662634978371dda8, thanks.

blueyed commented:
@timgraham 
Thanks!

Will this be back-ported to 1.8?

timgraham commented:
No, it doesn't qualify for a backport based on our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions).
